### PR TITLE
PWGJE: pT bin expanded for track residual

### PR DESCRIPTION
**Pull request by joonsukbae**

### CODE REVIEW

**Review by nzardosh — APPROVED** [review]

**Comment by joonsukbae** [comment]
I expanded pT range for both reco and gen level to remove pT smearing effect after unfolding, because statistics seem full enough to show until 200 GeV.
Also added mccollision counter to normalized generated jet distributions

**Comment by joonsukbae** [comment]
fixed some errors with pTbin settings and removed mccollision counter becuase it's already fully implemented in trackefficiency.cxx

File: PWGJE/Tasks/jetfinderhfQA.cxx [hunk 1]
```diff
@@ -194,122 +225,122 @@ struct JetFinderHFQATask {
...
+      registry.add("h3_jet_r_jet_pt_tag_jet_phi_base_diff_matchedgeo", "#it{R}_{jet};#it{p}_{T,jet}^{tag} (GeV/#it{c}); (#varphi_{jet}^{tag} - #varphi_{jet}^{base}) / #varphi_{jet}^{tag}", {HistType::kTH3F, {{jetRadiiBins, ""}, jetPtAxis, {1000, -5.0, 5.0}}});
+      registry.add("h3_jet_pt_tag_jet_eta_tag_jet_eta_base_matchedgeo", ";#it{p}_{T,jet}^{tag} (GeV/#it{c}); #eta_{jet}^{tag}; #eta_{jet}^{base}", {HistType::kTH3F, {jetPtAxis, {100, -1.0, 1.0}, {100, -1.0, 1.0}}});
+      registry.add("h3_jet_pt_tag_jet_phi_tag_jet_phi_base_matchedgeo", ";#it{p}_{T,jet}^{tag} (GeV/#it{c}); #varphi_{jet}^{tag}; #varphi_{jet}^{base}", {HistType::kTH3F, {jetPtAxis, {160, -1.0, 7.}, {160, -1.0, 7.}}});
+      registry.add("h3_jet_pt_tag_jet_ntracks_tag_jet_ntracks_base_matchedgeo", ";#it{p}_{T,jet}^{tag} (GeV/#it{c}); N_{jet tracks}^{tag}; N_{jet tracks}^{base}", {HistType::kTH3F, {jetPtAxis, {200, -0.5, 199.5}, {200, -0.5, 199.5}}});
+
+      registry.add("h3_jet_r_jet_pt_tag_jet_pt_base_matchedpt", "#it{R}_{jet};#it{p}_{T,jet}^{tag} (GeV/#it{c});#it{p}_{T,jet}^{base} (GeV/#it{c})", {HistType::kTH3F, {{jetRadiiBins, ""}, jetPtAxis, jetPtAxis}});
```
Comment (nzardosh) [inline PWGJE/Tasks/jetfinderhfQA.cxx:261]: for example here

**Comment by joonsukbae** [comment]
please check the latest modification

**Review by nzardosh — APPROVED** [review]

**Pull request merged**